src/components/Certify/Certify.vue

```diff
@@ -180,7 +220,7 @@ export default class Certify extends Vue {
   width: 100%;
 }
 
-.certify-clause {
+.certify-clause, .emails-line {
```

Comment:
What is this new class used for?

Reply:
typo indeed. Was renamed, good catch :)
 
Removed in next commit!